tests/test_ext_intersphinx.py

```diff
@@ -173,6 +181,7 @@ def test_missing_reference_pydomain(tempdir, app, status, warning):
         'https://docs.python.org/': inv_file,
     }
     app.config.intersphinx_cache_limit = 0
+    app.config.intersphinx_strict_prefix = False
```

Comment:
Is this initialization needed? I think the configuration is initialized as `False` by default. So this assignment is not needed. I understand setting it up explicitly in test_missing_reference. But I don't understand why are they added in other cases.

Reply:
I didn't work without that. Unfortunately, I have no clue how this works in the tests...